en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-soap-api-passthrough.md

```diff
@@ -0,0 +1,311 @@
+
+
+## Via apictl for Standalone Mode
+
+The WSO2 API Controller (apictl) does not provide a straight forward method for generating a `apictl` project for SOAP APIs. However you can use a project extracted from the WSO2 API Manager(you may download the project for API created in above guide) or a sample swagger for SOAP APIs. This guide will continue using a sample swagger for SOAP API.
+
+The swagger definition for our `PhoneVerify` example as follows.
+
+``` yaml
+info:
+  title: PhoneVerification
+  version: 1.0.0
+paths:
+  /*:
+    post:
+      consumes:
+      - text/xml
+      - application/soap+xml
+      parameters:
+      - description: SOAP request.
+        in: body
+        name: SOAP Request
+        required: true
+        schema:
+          type: string
+      - description: SOAPAction header for soap 1.1
+        in: header
+        name: SOAPAction
+        required: false
+        type: string
+      responses:
+        "200":
+          description: OK
+      security:
+      - default: []
+      x-wso2-application-security:
+        optional: false
+        security-types:
+        - oauth2
+security:
+- default: []
+securityDefinitions:
+  default:
+    authorizationUrl: https://test.com
+    flow: implicit
+    type: oauth2
+swagger: "2.0"
+x-wso2-application-security:
+  optional: false
+  security-types:
+  - oauth2
+x-wso2-auth-header: Authorization
+x-wso2-basePath: /phoneverify/1.0.0
+x-wso2-cors:
+  accessControlAllowCredentials: false
+  accessControlAllowHeaders:
+  - authorization
+  - Access-Control-Allow-Origin
+  - Content-Type
+  - SOAPAction
+  - apikey
+  - Internal-Key
+  accessControlAllowMethods:
+  - GET
+  - PUT
+  - POST
+  - DELETE
+  - PATCH
+  - OPTIONS
+  accessControlAllowOrigins:
+  - '*'
+  corsConfigurationEnabled: false
+x-wso2-production-endpoints:
+  type: http
+  urls:
+  - http://ws.cdyne.com/phoneverify/phoneverify.asmx
+x-wso2-response-cache:
+  cacheTimeoutInSeconds: 300
+  enabled: false
+x-wso2-sandbox-endpoints:
+  type: http
+  urls:
+  - http://ws.cdyne.com/phoneverify/phoneverify.asmx
+x-wso2-transports:
+- http
+- https
+```
+
+Following steps will show you how to create the `apictl` project, deploy it on the Choreo Connect gateway and invoke it using a test token, for our `PhoneVerify` example.
+
+!!! Info "Before you begin"
+    Please make sure you have completed [Download and initilaize]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool) `apictl` and Choreo Connet is up and running in [standalone mode]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/getting-started/deploy/cc-deploy-overview/#choreo-connect-deployment-options). 
+
+
+### Step 1 - Initialize the apictl project
+
+Use the follwing command to initialize the `apictl` project. You may use the above sample swagger provided.
+
+```bash
+apictl init PhoneVerify --oas <path to swagger.yaml>
+```
```

Comment:
We can point to a sample soap api project which is hosted in product-microgateway samples folder.

Reply:
Added a tip, saying that we can use the project in samples directory to continue with the same steps.